arcaflow_plugin_horreum_client/horreum_client_plugin.py

```diff
@@ -61,9 +71,19 @@ def horreum_client(
             verify=params.tls_verify,
         )
 
-    except (requests.ConnectionError, requests.HTTPError, requests.Timeout):
+    except requests.exceptions.SSLError as e:
+        return "error", ErrorOutput(
+            f"Unable to establish SSL connection with the Horreum server: {e}"
+        )
+
+    except (requests.ConnectionError, requests.Timeout) as e:
+        return "error", ErrorOutput(
+            f"Error communicating with the Horreum server: {e}"
+        )
+
+    except requests.HTTPError:
```

Comment:
That's dead code, since you're not using `raise_for_exception` here. In fact, if you ever *did* get here, the `auth_return` reference would have to be a failure. (And I believe that `HTTPError` actually has a `response` field giving access to Request's response object directly.)